### ISSUE TITLE
Could not resolve airline theme ubaryd

### ISSUE THREAD

**Issue by Tekki** [vim-airline/vim-airline-themes#162]
Since update from ad6cc5 (2018-11-08) to 011960 (2018-11-12) I get the following error when I start Vim:

> Could not resolve airline theme "ubaryd". Themes have been migrated to github.com/vim-airline/vim-airline-themes.

`:AirlineTheme ubaryd` works as expected.
Vim 8.1.477 and 8.1.526 on Debian Stretch, plugins installed at ~/.vim/pack/plugins/start

**Comment by Tekki** [comment]
The error appears with update from ad6cc5 to dad5bc (2018-11-09).

**Comment by chrisbra** [comment]
ping @Donearm 

There are some serious problems with your newly created themes.

ubaryd uses some variables that are not even defined, like `s:N4`, `s:V4` and `s:RE`.

Also it needs an entry `airline_c` in the dict `g:airline#themes#ubaryd#palette.inactive` since this is needed by the tabline section.

Same for the theme laederon, plus there is a quote too much:

```patch
iff --git a/autoload/airline/themes/laederon.vim b/autoload/airline/themes/laederon.vim
index 69555a2..b26a16a 100644
--- a/autoload/airline/themes/laederon.vim
+++ b/autoload/airline/themes/laederon.vim
@@ -38,7 +38,7 @@ let g:airline#themes#laederon#palette.replace = airline#themes#generate_color_ma
 let g:airline#themes#laederon#palette.inactive = airline#themes#generate_color_map(s:IA, s:IA, s:IA)

 " Warning Mode
-let s:WI = [ '#242321' , '#ab3e5b'' , 235 , 161 , '' ]
+let s:WI = [ '#242321' , '#ab3e5b' , 235 , 161 , '' ]
 let g:airline#themes#laederon#palette.normal.airline_warning = [
         \ s:WI[0], s:WI[1], s:WI[2], s:WI[3]
         \ ]
```

**Comment by Donearm** [comment]
thanks @Tekki for reporting.

Does Donearm/vim-airline-themes@554479b fix it?
Do you have also installed the Ubaryd and/or Laederon vim themes (not airline)?

**Comment by Tekki** [comment]
@Donearm Yes, the code from your testing branch fixes it. :cocktail: :cocktail: :cocktail:
I've installed Ubaryd and Laederon since 10 minutes (didn't know they existed, just selected the ubaryd color for airline because it looked the best). Are you prepared for more issues?

**Comment by Donearm** [comment]
Glad it fixed :smile:

I'd happily do without any more issues but eventually coffee and laptop are ready :+1: 

@chrisbra shall I open a new PR? Or will you merge in the old one?

**Comment by chrisbra** [comment]
please go ahead and create a new PR. Thanks

**Comment by chrisbra** [comment]
closed by aea73d5
